### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by markoninho**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
 # sky9
 AWS accounts and users management
+
+## Unit Testing
+ Run the test with # npm test command
```
Comment (niktcha) [inline README.md]: ```npm test``` instead of description, use triple backtick (```) around the code.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
 # sky9
 AWS accounts and users management
+
+## Unit Testing
+ Run the test with # npm test command 
+
+## Deploy
+
```
Comment (niktcha) [inline README.md]: extra line

File: README.md [hunk 3]
```diff
@@ -1,2 +1,43 @@
...
+```sh
+    sls dynamodb start 
+```
+Run serverless offline:
+```sh
+    sls offline
+```
+Run ESLint:
+```sh
+    esLint .
```
Comment (mjovanovic) [inline README.md]: Do you need to have it globally installed?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,43 @@
 # sky9
 AWS accounts and users management
+
```
Comment (mjovanovic) [inline README.md:11]: Please mention the prerequisites for all the things below. Like: NodeJS, Serverless, configured AWS account.

File: README.md (hunk 1 above)
Comment (markoninho) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (markoninho) [reply README.md]: Yes, we need it.

Comment (markoninho) [reply README.md]: It's added to documentation.

File: README.md (hunk 4 above)
Comment (markoninho) [reply README.md:11]: Done.

**Review by niktcha — APPROVED** [review]

**Pull request merged**